app/services/reports/period_supply_report_service.rb

```diff
@@ -137,7 +137,7 @@ def kit_items_calculation(itemizable_type, string_itemizable_type)
         INNER JOIN base_items ON base_items.partner_key = kit_items.partner_key 
         WHERE #{itemizable_type}.organization_id = ?
           AND EXTRACT(year FROM issued_at) = ?
-          AND LOWER(base_items.category) LIKE '%menstral supplies%'
+          AND LOWER(base_items.category) LIKE '%menstrual  supplies%'
```

Comment:
You've got two spaces instead of one between menstrual and supplies.    I checked that out  on local, and it makes my initial test work for the Period supplies distributed.